src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge31.java

```diff
@@ -0,0 +1,90 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import java.nio.charset.StandardCharsets;
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
+import java.util.UUID;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.owasp.wrongsecrets.RuntimeEnvironment.Environment.DOCKER;
+
+@Component
```

Comment:
Can you please add the required javadoc :) ? (see documentation/other challenges for example)

Reply:
yep sure

Reply:
Fixed it in a commit, thanks!

Reply:
I'm really sorry @commjoen 
I'm kinda preoccupied with another task at https://github.com/puneeth072003/UtopiaDiscordbot
I couldn't find much time to contribute here sorry.

Reply:
i need some time to add javadoc